.github/workflows/sonar.yml

```diff
@@ -0,0 +1,37 @@
+name: Build
```

Comment:
it more like Test workflow, or Build and test

Reply:
Nice catch, updated, thanks!